pages/post/[id]/index.tsx

```diff
@@ -19,15 +23,28 @@ const PostPage: NextPage = () => {
   })
   const { loading: userLoading, error: userError, data: userData } = useCurrentUserQuery()
 
+  const post: PostType | undefined | null = postData?.postById
```

Comment:
I'm definitely happy to see the null check lifted out of the `Post` component, maybe I missed it but did we update `Post` to not allow `null` or `undefined` values for the `post` prop, and remove null checking around that?

Reply:
DONE! ✅